lametro/views.py

```diff
@@ -797,3 +799,16 @@ def metro_login(request):
 def metro_logout(request):
     logout(request)
     return HttpResponseRedirect('/')
+
+@csrf_exempt
+def refresh_guid_trigger(request, refresh_key):
+    try:
+        if refresh_key == settings.REFRESH_KEY:
+            management.call_command('refresh_guid')
+            return HttpResponse(200)
+    except AttributeError:
+        if not refresh_key:
+            print('You need a refresh_key in your local deployment settings files to access this.')
+        else:
```

Comment:
good instinct, but it would be more appropriate to pair this `else` with the `if` statement checking whether the given key is correct.